node/node.go

```diff
@@ -318,8 +318,8 @@ func (n *Node) ProcessBatch(ctx context.Context, header *core.BatchHeader, blobs
 		// revert all the keys for that batch.
 		result := <-storeChan
 		if result.keys != nil {
-			if !n.Store.DeleteKeys(ctx, result.keys) {
-				log.Error("Failed to delete the invalid batch that should be rolled back", "batchHeaderHash", batchHeaderHash)
+			if err = n.Store.DeleteKeys(ctx, result.keys); err != nil {
```

Comment:
assign to a different err variable? The `err` logged in L322 should be the err in L315, but this overrides that value